corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -178,10 +178,14 @@ hqDefine('registration/js/new_user.ko.js', function () {
         self.currentStep = ko.observable(0);
 
         var _getDataForSubmission = function () {
+            var password = self.password()
+            if(typeof(hex_parser) != 'undefined') {
+                var password = (new hex_parser()).encode(self.password())
```

Comment:
'password' is already defined (no-redeclare)
'hex_parser' is not defined. (no-undef)
Missing semicolon. (semi)